### PR TITLE
Add table lines in latex format in mustache file

### PR DESCRIPTION
**Pull request by ameliav**
Adds hardcoded text into each levels table of the Templates section of Appendix C.

## 🗣 Description ##

“CISA average click rate for level” entry added to lines: 1632, 1655, 1679, 1702, 1725, 1748.
“CISA average report rate for level” entry added to lines: 1636, 1659, 1683, 1706, 1729, 1752. 
PCADEV-111 requested by Kelly Thiele.

## 💭 Motivation and context ##

Not a code change. This will add latex formatted table lines to the mustache file with CISA average click and report rates for each level within the Templates section of Appendix C. 

## 🧪 Testing ##

N/A

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.

### CODE REVIEW

**Review by BenBreaksThings — APPROVED** [review]
Purely text template updates as requested

**Review by dav3r — COMMENTED** [review]
👎  Boo to hardcoded data values!  I'm assuming you are doing this until these values can be calculated programatically?  If so, I'd recommend adding a comment by the first place you are doing this and explaining why this is being done.  I also strongly recommend creating an issue in this repo to document that these values should be replaced later with calculated values.

**Comment by jsf9k** [comment]
Additionally, PR titles and the first line of git commit messages should be in [the imperative mood](https://en.wikipedia.org/wiki/Imperative_mood), e.g. prefer "Add table lines in latex format" to "Adds table lines in latex format".

For more on writing quality git commit logs please see [here](https://chris.beams.io/posts/git-commit/#seven-rules), which is referenced in [our development guide](https://github.com/cisagov/development-guide#git-github-and-you).

**Comment by ameliav** [comment]
They are values that change yearly and I have not been given a way to automatically change them, I was told to add them this way.

**Comment by ameliav** [comment]
At this point, I'm not sure that they can be programmatically be changed. They are values I got from Kelly T. and they would always come from outside the repo (Averages per year). But they will need to be updated on a yearly basis. Open to any suggestions

**Comment by BenBreaksThings** [comment]
Can corroborate @ameliav 's assessment on the values. The values are annually determined, will be updated in this repo only once annually, and have long been a missing/desired data point within our reports. Hard coding is not ideal, but these values will not be changing and will not go unnoticed in the reports that get picked apart numerous times per quarter. This upcoming Tuesday at the Fusion dev sync PCA wanted to discuss the DB situation again, wherein we could explore what is within the realm of the possible for in the future just pulling this stuff from the DB, but that is likely 6+ months from making its way into this repo.

**Comment by ameliav** [comment]
This PR is ready for review.

File: src/customer/report.mustache [hunk 1]
```diff
@@ -1624,14 +1624,17 @@ Below are the details about the templates used during PCA with <<Acronym>>. The
 \end{center}
 \hangindent1em \textbf{Key characteristics:} <<Level-1-Key_Characteristics>>
 
+%CISA averages rates should be updated yearly with previous year's average rates. Current rates are for 2020.
```
Comment (dav3r) [inline src/customer/report.mustache]: For readability, we prefer comments with a space after the comment character and keeping lines shorter than 80 characters whenever possible.  Also, improve grammar:
```suggestion
% CISA average rates should be updated yearly with previous year's average rates.
% Current rates are for 2020.
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Please see the one small change that I requested.

**Review by dav3r — APPROVED** [review]
Looks good! 👍

**Pull request merged**